packages/odbc/install

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# fetch install script
+wget https://drivers.rstudio.org/7C152C12/odbc-install.sh
+chmod +x odbc-install.sh
+mkdir /etc/rstudio-drivers
+
+# install & configure drivers
+./odbc-install.sh --install /etc/rstudio-drivers --eula
```

Comment:
Please don't install into `/etc`, `/opt` is far more appropriate

Reply:
Definitely makes sense.  `/etc` is where we seem to install / recommend installing everything (drivers, RSP, RSC, etc.), but I agree.

Reply:
`/etc` is where configuration for things go, but it is not where we recommend installing the software itself.

Reply:
Ahhhh.  That makes sense.  My mistake!